### ISSUE TITLE
Make blueflower pip installable

### ISSUE THREAD

**Issue by guykisel** [veorq/blueflower#3]
I just discovered blueflower and wanted to give a try (it looks like a pretty cool tool!), but I was surprised to find it doesn't seem to be available on https://pypi.python.org/pypi

**Comment by veorq** [comment]
Yeah this is missing, sorry. I'll probably add, but at the moment that's not on the top of my todo list.

**Comment by guykisel** [comment]
It's really just a nice-to-have thing, it doesn't strike me as being urgently needed. Cloning the repo isn't that bad :)
